tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1648,45 +1704,35 @@ def test_continuous_deletes_during_pod_restart(self):
         assert_utils.assert_equal(len(buckets), wr_bucket, f"Failed to create {wr_bucket} number "
                                                            f"of buckets. Created {len(buckets)} "
                                                            f"number of buckets")
-        LOGGER.info("Step 5: Successfully performed WRITEs with variable object sizes. (0B - "
-                    "128MB)")
-
-        LOGGER.info("Step 6: Verify %s has %s buckets created", self.s3acc_name, wr_bucket)
+        LOGGER.info("Step 3: Successfully performed WRITEs with variable object sizes.")
+        LOGGER.info("Step 4: Verify %s has %s buckets created", self.s3acc_name, wr_bucket)
```

Comment:
Do we need this step? We are already verifying the same in above step assert_utils.assert_equal(len(buckets), wr_bucket,

Reply:
yeah u r right, we might ignore this tep